app.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-unused-vars */
+const menuLinks = document.getElementById('menuChild');
+const hamburger = document.getElementById('menuIcon');
+const maincontent = document.getElementById('mobmain');
+const logo = document.getElementById('logo');
+const openMenu = document.querySelector('.openMenu');
+const closeMenu = document.querySelector('.closeMenu');
+const navlinks = document.querySelectorAll('.a');
+const seeproject = document.querySelectorAll('.Button-SeeProject');
+const desktopMenu = document.querySelector('.desktopMenu');
+const modal = document.querySelector('#modal');
+const projectImage = document.querySelector('.idcard-modal');
+const projectTitle = document.querySelector('.modal-tonic');
+
+openMenu.addEventListener('click', (() => {
+  menuLinks.style.display = 'flex';
+  menuLinks.style.visibility = 'visible';
+  hamburger.style.display = 'none';
+  maincontent.classList.add('blur');
+  logo.classList.add('blur');
+}));
+
+closeMenu.addEventListener('click', (() => {
+  menuLinks.style.display = 'none';
+  menuLinks.style.visibility = 'hidden';
+  hamburger.style.display = 'flex';
+  maincontent.classList.remove('blur');
+  logo.classList.remove('blur');
+}));
+
+for (let i = 0; i < navlinks.length; i += 1) {
+  navlinks[i].addEventListener('click', () => {
+    maincontent.classList.remove('blur');
+    logo.classList.remove('blur');
+    menuLinks.style.display = 'none';
+    menuLinks.style.visibility = 'hidden';
+    hamburger.classList.remove('blur');
+    hamburger.style.display = 'flex';
+  });
+}
+
+/// /////////cards/////////
+
+const storeData = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+  {
+    name: 'Multi-post',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'desktop-2.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'desktop-3.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+  {
+    name: 'Multi-post',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'desktop-4.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
```

Comment:
While you did a wonderful job styling the popup 👍, kindly you should make sure that your portfolio match as much as possible to the Figma template.

[REQUIRED] - As Suggested by the previous code reviewer. There is an empty space at the end of the popup that should be removed. it would be nice to stick to the figure designs please. You can fix this issue by reducing the height of the pop-up window lets pay attention to detail.
 
This is your mobile pop-up design:
![223728977-40570e82-d32f-4651-b794-d01bd1503086](https://user-images.githubusercontent.com/79506105/223749817-68a7f7b3-b7d1-401a-aaa0-a17571858bbb.png)


Please stick to this Figma design:
![223729064-5c2db3f1-1168-4115-a7e6-9559a26afc97](https://user-images.githubusercontent.com/79506105/223749965-ea3a983e-44b0-45a4-a84e-1f9825ae6f72.png)


[REQUIRED]Secondly as the previous code reviewer suggested it would be nice if you  please ensure that the desktop version popup is sized to fit on the same page without requiring a scrollbar
this means then the pop window is opened it should be fixed and all its content should be seen with scrolling up or down.

This is your design:
![223730371-ca230dc6-6bd3-4dcb-ba1b-7e392cc2b739](https://user-images.githubusercontent.com/79506105/223750815-cf919486-5a3e-405d-8fdb-595dbfd5133c.png)

Your pop up window should look like this figma design:
![Screenshot 2023-03-08 at 5 14 00 PM](https://user-images.githubusercontent.com/79506105/223751393-27976b85-7286-40e2-9eab-e23d1a858ed0.png)

Please pay attention to detail to avoid making mistakes.